openshift/helm/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: quality-time
```

Comment:
```suggestion
name: Quality-time
```

Reply:
The name is based on the naming convention for Helm charts: https://helm.sh/docs/chart_best_practices/conventions/